docs/user-guide/address-security-requirements.md

```diff
@@ -0,0 +1,73 @@
+## Security Permissions Reference Table 
+
+The following reference table describes which permissions are required for the user ID `ZWESVUSR` to run Zowe core services and specific individual components.
+
+If you already successfully ran the [`ZWESECUR`](/docs/user-guide/initialize-security-configuration.md#configuring-with-zwesecur-jcl) JCL either separately or by running the [`zwe init security`](/docs/user-guide/initialize-security-configuration.md#configuring-with-zwe-init-security-command) command, you do not need to perform the steps described in this section. The TSO commands to create the user IDs and groups are executed during the JCL sections of `ZWESECUR`. For more information about the `zwe init security` command, see [zwe init security](../appendix/zwe_server_command_reference/zwe/init/zwe-init-security.md).
+
+| Feature of a Zowe server-side component                    | Resource class    | Resource name                          | Type of access required | Reason | Actions                                                                                                                                                                                                                        |
+|------------------------------------------------------------|----------|-----------------------------|--------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----|
+| Cross memory server (ZIS)                                  | CSFSERV                                 | `Multiple`                  | READ   | Generate symmetric keys using ICSF that is used by [Zowe Desktop cookies](./configure-zos-system.md#configure-an-icsf-cryptographic-services-environment). | The list of IDs to enable include `CSF1TRD` , `CSF1TRC` , `CSF1SKE` , `CSF1SKD`. The full list of IDs is described in the z/OS Cryptographic Services user guide for your z/OS release level: [2.2](https://www.ibm.com/docs/en/zos/2.2.0?topic=ssl-racf-csfserv-resource-requirements), [2.3](https://www.ibm.com/docs/en/zos/2.3.0?topic=ssl-racf-csfserv-resource-requirements), [2.4](https://www.ibm.com/docs/en/zos/2.4.0?topic=ssl-racf-csfserv-resource-requirements) and [2.5](https://www.ibm.com/docs/en/zos/2.5.0?topic=ssl-racf-csfserv-resource-requirements). |
```

Comment:
I would not lable it core since many customers are using Zowe without ZSS/ZIS. 

Core should be permissions that are mandatory and zowe will not run without them.

Reply:
Are you sure that Zowe will not use CSF for things like generating and reading keyrings?
I'm not sure, I think it might, so that's why I said core.

Reply:
Keyrings are generated under a security admin ID. This table describes permission needed from a Zowe server user.